source/clear-linux/guides/telemetrics/telem-guide.rst

```diff
@@ -356,7 +356,31 @@ There are three ways to supply the payload to the record.
 
       payload read from stdin
 
+Set a static machine id
+=======================
+
+The machine id reported by the telemetry client is rotated every three days
+for privacy reasons. If you wish to have a static machine id for testing
+purposes, you can opt in by creating a file named "opt-in-static-machine-id"
+in the directory  :file:`/etc/telemetrics/`.
+
```

Comment:
@bd-dean Would you advise running: `hostnamectl` here beforehand?

Reply:
I would not recommend :command:`hostnamectl` because it will not report the change to the machine-id, so the user may be worried that telemctl did not pick up their changes.

Systemd and hostnamectl are reading the /etc/machine-id file to pick up & report machine-id. telemctl overrides with the name in the opt-in file but the /etc/machine-id file isn't changed.

Reply:
Fair enough. Thanks for the detailed response. Any other changes, or are you ready to merge?

Reply:
Good to go!